src/Providers/LIAuthServiceProvider.php

```diff
@@ -0,0 +1,60 @@
+<?php
+
+namespace LiveIntent\LaravelCommon\Providers;
+
+use Illuminate\Auth\RequestGuard;
+use Illuminate\Support\Facades\Auth;
+use LiveIntent\LaravelCommon\Auth\Guards\LITokenGuard;
+use Illuminate\Foundation\Support\Providers\AuthServiceProvider as ServiceProvider;
+
+class LIAuthServiceProvider extends ServiceProvider
+{
+    public function __construct($app)
+    {
+        parent::__construct($app);
+        $this->registerEnvironmentVariables();
+    }
+
+    /**
+     * Register any authentication / authorization services.
+     *
+     * @return void
+     */
+    public function boot()
+    {
+        $this->registerCustomGuards();
+        $this->registerPolicies();
+    }
+
+    protected function registerEnvironmentVariables()
+    {
+        if (config('auth.li_token.keys.public') == null) {
+            config(['auth.li_token.keys.public' => env('TESSELLATE_PUBLIC_KEY') ]);
+        }
+
+        if (config('auth.guards.li_token') == null) {
+            config(['auth.guards.li_token' => [
+                'driver' => 'li_token',
+                'provider' => 'users',
+            ]]);
+        }
+
+        $this->policies = config('auth.li_token.policies', []);
```

Comment:
Same recurring question I guess, why here have we introduced a new way of defining the policies?

Reply:
This was done so that the `AuthServiceProvider` didn't need to be extended in order to add support for policies in Liam. If we want to let it continue to require the class to be extended, I can revert this change